error tracking q2 goals (#11030)
PostHog/posthog.com: contents/teams/error-tracking/objectives.mdx

### Q2 2025 Objectives

#### Launch with pricing
* Decide on tiers and implement billing changes
* Remove plugin server from ingestion pipeline (realise cost savings)
* Work with comms on rollout

> #### Make it easier to integrate

Author: ```suggestion
#### Make it easier to integrate
* Split stack trace parsing from exception autocapture
```

### Q2 2025 Objectives

#### Launch with pricing
* Decide on tiers and implement billing changes
* Remove plugin server from ingestion pipeline (realise cost savings)
* Work with comms on rollout

#### Make it easier to integrate
* CLI download URL that points to the latest version
* Add more source map upload options
    * GitHub action
    * NextJS config wrapper
    * Bundler plugins for at least Webpack and Vite
* Stretch: Isometric SDK for NextJS

#### Product quality

> * Performance

Author: ```suggestion
```

### Q2 2025 Objectives

#### Launch with pricing
* Decide on tiers and implement billing changes
* Remove plugin server from ingestion pipeline (realise cost savings)
* Work with comms on rollout

> #### Make it easier to integrate

Reviewer: Make autocapture not necessary for stack traces - we should _always_ capture stack traces

* CLI download URL that points to the latest version
* Add more source map upload options
    * GitHub action
    * NextJS config wrapper
    * Bundler plugins for at least Webpack and Vite
* Stretch: Isometric SDK for NextJS

#### Product quality

> * Performance

Reviewer: Cut this - it's one element of quality relevant to ALL other work

Author: Added it under issue page improvements to improve the loading time

> * UI improvements to issue page
> * Intentionally gather more feedback from customers
>     * Measure grouping signal / noise ratio and impact of the problem
>     * Ensure each small team in PostHog is setup to use error tracking
> * Stretch: Omnisearch

Author: ```suggestion
* UI improvements to issue page
    * Faster initial loading time (separate events query for latest / earliest event)
    * Highlight person and groups
    * Stretch: 
* Measure grouping signal / noise ratio and impact of the problem
```

### Q2 2025 Objectives

### Launch with pricing
* Decide on tiers and implement billing / quota limiting changes
* Remove plugin server from ingestion pipeline (cost savings)
* Work with comms on rollout

### Be the default choice for the languages we support

#### Make it easier to integrate
* Split stack trace parsing from exception autocapture
* CLI download URL that points to the latest version
* Add more source map upload options
    * GitHub action
    * NextJS config wrapper
    * Bundler plugins for at least Webpack and Vite

> * Stretch: Isometric SDK for NextJS

Reviewer 2: ```suggestion
* Stretch: Isomorphic SDK for NextJS
```

* UI improvements to issue page
* Intentionally gather more feedback from customers
    * Measure grouping signal / noise ratio and impact of the problem
    * Ensure each small team in PostHog is setup to use error tracking
* Stretch: Omnisearch

#### Build advanced features for larger customers

> * Support tags for issue

Author: ```suggestion
```
